packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -127,6 +127,29 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
     return t;
   });
 
+  const splitURLPath = useMemo(() => location.pathname.split('/'), []);
+  const decodedString = useMemo(
+    () => decodeURIComponent(splitURLPath[3]),
+    [splitURLPath],
+  );
+
+  //redirects users to All Discussions if they try to access a topic in the url that doesn't exist
+  useEffect(() => {
+    if (
+      !isLoadingTopics &&
+      decodedString &&
+      splitURLPath[2] === 'discussions' &&
```

Comment:
Did we test this piece against custom domains?

Reply:
@jnaviask This should be fine for custom domains as the url is being split on "/", so there should be no difference.

Reply:
in https://gov.1inch.io/discussions 
![image](https://github.com/user-attachments/assets/4c3e1d3a-d89a-436e-9fd9-45559bebb3a5)


in https://commonwealth.im/1inch/discussions
![image](https://github.com/user-attachments/assets/387eb72d-6ff5-4b37-9286-d16446453a6a)

Reply:
Whoops. Okay, I was wrong, I will fix that.